### DISCUSSION TITLE
How to hide hidden files (like .DS_Store) from web ui?

### DISCUSSION THREAD

**Discussion by cesboa** [drakkan/sftpgo#1688] [Q&A]
Is there a nice way to hide or exclude files form the web ui? I looked thought the config and couldn't find anything regarding this. It could be done at the front end with a bit of js, but that is not as nice as the files not being served in the first place.
What would be the best practice for this?

Cheers,

![Screenshot 2024-07-05 at 09 33 52](https://github.com/drakkan/sftpgo/assets/1929823/56913e65-6f87-45a6-b4b1-b89ac5594b58)

**Comment by cesboa** [comment]
Used User Group ACL to exclude the .ds_store

**Comment by theturtle32** [comment]
What about for when you are making a shared folder?
